token-swap/js/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "description": "SPL Token Swap JavaScript API",
   "license": "MIT",
-  "author": "Solana Maintainers <maintainers@solana.com>",
+  "author": "Solana Maintainers <maintainers@solana.foundation>",
```

Comment:
Is this correct? I noticed the packages in the monorepo used this address

Reply:
yep!  it's correct